test/authentication_tests.coffee

```diff
@@ -73,10 +77,11 @@ Meteor.startup ->
         test.equal response.status, 'success'
         next()
 
-    it 'should remove the logout token after logging out', (test, next) ->
+    it 'should remove the logout token after logging out and should respond after 500 msec', (test, next) ->
```

Comment:
I think it's safe to test for the delay just once. We can just let the delay happen here. We've already verified it works above.